mkt/latecustomization/migrations/0001_initial.py

```diff
@@ -0,0 +1,35 @@
+                ('carrier', models.IntegerField(choices=[(15, mkt.constants.carriers.TELENOR), (7, mkt.constants.carriers.KDDI), (11, mkt.constants.carriers.SINGTEL), (13, mkt.constants.carriers.SPRINT), (14, mkt.constants.carriers.TELECOM_ITALIA_GROUP), (17, mkt.constants.carriers.VIMPELCOM), (9, mkt.constants.carriers.MEGAFON), (2, mkt.constants.carriers.AMERICA_MOVIL), (6, mkt.constants.carriers.HUTCHINSON_THREE_GROUP), (19, mkt.constants.carriers.CONGSTAR), (22, mkt.constants.carriers.ORANGE), (21, mkt.constants.carriers.MTN), (0, mkt.constants.carriers.UNKNOWN_CARRIER), (20, mkt.constants.carriers.O2), (12, mkt.constants.carriers.SMART), (5, mkt.constants.carriers.ETISALAT), (18, mkt.constants.carriers.GRAMEENPHONE), (8, mkt.constants.carriers.KT), (10, mkt.constants.carriers.QTEL), (3, mkt.constants.carriers.CHINA_UNICOM), (1, mkt.constants.carriers.TELEFONICA), (16, mkt.constants.carriers.TMN), (4, mkt.constants.carriers.DEUTSCHE_TELEKOM)])),
```

Comment:
We could make a custom field that doesn't serialize this info in the migrations.